react/week4/githubuser/src/GithubRepos.js

```diff
@@ -0,0 +1,31 @@
+import React, { useContext } from "react";
+import { UsersContext } from "./UsersContext";
+
+const GitHubRepos = () => {
+  const repoContext = useContext(UsersContext);
+  //fetching the github repos
+  const getGitHubRepos = () => {
+    fetch(`https://api.github.com/users/HackYourFuture-CPH/repos`)
+      .then((res) => res.json())
+      .then((reposdata) => {
+        repoContext.setRepos(reposdata);
+      });
+  };
+
+  return (
+    <>
+      <h1>HackYour Future Repositories</h1>
+      <div className="githubrepos">
+        <a href="#" onClick={getGitHubRepos}>
+          GitHub Repos
+        </a>
+        <ul>
+          {repoContext.repos &&
+            repoContext.repos.map((repo) => <li key={repo.id}>{repo.name}</li>)}
```

Comment:
If you want to simplify this one, you can use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)
This will allow you to check if the repos exist in the same line as you `.map` them